_includes/releases/cloud/2023-05-01.md

```diff
@@ -0,0 +1,38 @@
+## May 1, 2023
+
+<h3> General changes </h3>
+
+The following changes were made to {{ site.data.products.serverless }} for **new organizations** created on or after April 26, 2023. **Existing organizations will not be affected until August 1, 2023**.
+
+- The price of [Request Units (RUs)](../cockroachcloud/learn-about-request-units.html) increased from $1 per 10M RU to $2 per 10M RUs.
+- The price of [storage](../cockroachcloud/architecture.html#storage) decreased from $1 per 1 GiB storage to $0.50 per 1 GiB storage.
+- Free resources are allocated on a per-organization basis instead of a per-cluster basis. All organizations will now receive 50M Request Units per month and 10 GiB of storage for free.
+- All resources are available instantly at the beginning of each month (burst and baseline RUs are now deprecated).
+- You can now [set separate RU and storage limits](../cockroachcloud/serverless-cluster-management.html#edit-your-resource-limits) for your clusters, or set a spend limit that will be divided between resources automatically.
+- You can now [set unlimited resources](../cockroachcloud/serverless-cluster-management.html#edit-your-resource-limits) for a cluster so your workload is never restricted.
+- Organizations on the free plan will have at most one free Serverless cluster (instead of the previous limit of five free clusters). **We will continue to maintain all existing free clusters with the new organization-level free resources**. 
```

Comment:
```suggestion
- Organizations on the free plan will have at most one free serverless cluster (instead of the previous limit of five free clusters). **Cockroach Labs will continue to maintain all existing free clusters with the new organization-level free resources**. 
```